packages/java/tests/spring/react-i18n/frontend/_views/BasicI18NView.tsx

```diff
@@ -0,0 +1,20 @@
+import { i18n, translate } from '@vaadin/hilla-react-i18n';
```

Comment:
The package here is `_views` instead of just `views`. The reason is to align with the similar temporary changes in the other test modules.